bundles/org.openhab.binding.evcc/README.md

```diff
@@ -68,17 +126,175 @@ Bridge evcc:server:demo-server "Demo" [scheme="https", host="demo.evcc.io", port
     Thing loadpoint demo-loadpoint-carport "Loadpoint - evcc Demo Loadpoint 1"[index=0]
     ..
     // You can define as many Vehicle things as you have vehicles configured in your evcc instance
-    Thing vehicle demo-vehicle1 "Vehicle - evcc Demo Vehicle 1"[id="vehicle_1"]
+    Thing vehicle demo-vehicle1 "Vehicle - evcc Demo Vehicle 1"[vehicle-id="vehicle_1"]
+    ..
+    // You can define as many Plan things as you have plans for your vehicle configured
+    Thing plan demo-one-time-plan-for-vehicle1 "One-time plan for vehicle 1"[index=0, vehicle-id="vehicle_1"]
+    Thing plan demo-repeating-plan-1-for-vehicle1 "Repeating plan 1 for vehicle 1"[index=1, vehicle-id="vehicle_1"]
     ..
 }
 ```
 
 ### `demo.items` Example
 
+List of all possible channels
+
 ```java
-Number GridPower "Grid Power" { channel="evcc:site:demo-server:demo-site:site-grid-power" }
+Number:Energy Evcc_Battery_Capacity "Battery Capacity [%s]" { channel="evcc:battery:demo-server:demo-battery:Battery_Capacity" }
```

Comment:
Would it be possible to align this list in columns by making use of spacing between type, id, label and {} part ?

Reply:
Ah, yeah sorry for the misunderstanding.